compiler/ballerina-backend-jvm/src/main/ballerina/compiler_backend_jvm/jvm_constants.bal

```diff
@@ -0,0 +1,195 @@
+const int L2I = 136; // -
+const int L2F = 137; // -
+const int L2D = 138; // -
+const int F2I = 139; // -
+const int F2L = 140; // -
+const int F2D = 141; // -
+const int D2I = 142; // -
+const int D2L = 143; // -
+const int D2F = 144; // -
+const int I2B = 145; // -
+const int I2C = 146; // -
+const int I2S = 147; // -
+const int LCMP = 148; // -
+const int FCMPL = 149; // -
+const int FCMPG = 150; // -
+const int DCMPL = 151; // -
+const int DCMPG = 152; // -
+const int IFEQ = 153; // visitJumpInsn
+const int IFNE = 154; // -
+const int IFLT = 155; // -
+const int IFGE = 156; // -
+const int IFGT = 157; // -
+const int IFLE = 158; // -
+const int IF_ICMPEQ = 159; // -
+const int IF_ICMPNE = 160; // -
+const int IF_ICMPLT = 161; // -
+const int IF_ICMPGE = 162; // -
+const int IF_ICMPGT = 163; // -
+const int IF_ICMPLE = 164; // -
+const int IF_ACMPEQ = 165; // -
+const int IF_ACMPNE = 166; // -
+const int GOTO = 167; // -
+const int JSR = 168; // -
+const int RET = 169; // visitVarInsn
+const int TABLESWITCH = 170; // visiTableSwitchInsn
+const int LOOKUPSWITCH = 171; // visitLookupSwitch
+const int IRETURN = 172; // visitInsn
+const int LRETURN = 173; // -
+const int FRETURN = 174; // -
+const int DRETURN = 175; // -
+const int ARETURN = 176; // -
+const int RETURN = 177; // -
+const int GETSTATIC = 178; // visitFieldInsn
+const int PUTSTATIC = 179; // -
+const int GETFIELD = 180; // -
+const int PUTFIELD = 181; // -
+const int INVOKEVIRTUAL = 182; // visitMethodInsn
+const int INVOKESPECIAL = 183; // -
+const int INVOKESTATIC = 184; // -
+const int INVOKEINTERFACE = 185; // -
+const int INVOKEDYNAMIC = 186; // visitInvokeDynamicInsn
+const int NEW = 187; // visitTypeInsn
+const int NEWARRAY = 188; // visitIntInsn
+const int ANEWARRAY = 189; // visitTypeInsn
+const int ARRAYLENGTH = 190; // visitInsn
+const int ATHROW = 191; // -
+const int CHECKCAST = 192; // visitTypeInsn
+const int INSTANCEOF = 193; // -
+const int MONITORENTER = 194; // visitInsn
+const int MONITOREXIT = 195; // -
+const int MULTIANEWARRAY = 197; // visitMultiANewArrayInsn
+const int IFNULL = 198; // visitJumpInsn
+const int IFNONNULL = 199; // -
+
+
+// asm constants
+const int ACC_PUBLIC = 1;
+const int ACC_PRIVATE = 2;
+const int ACC_PROTECTED = 4;
+const int ACC_STATIC = 8;
+const int ACC_FINAL = 16;
+const int ACC_SUPER = 32;
+const int ACC_SYNCHRONIZED = 32;
+const int ACC_OPEN = 32;
+const int ACC_TRANSITIVE = 32;
+const int ACC_VOLATILE = 64;
+const int ACC_BRIDGE = 64;
+const int ACC_STATIC_PHASE = 64;
+const int ACC_VARARGS = 128;
+const int ACC_TRANSIENT = 128;
+const int ACC_NATIVE = 256;
+const int ACC_INTERFACE = 512;
+const int ACC_ABSTRACT = 1024;
+const int ACC_STRICT = 2048;
+const int ACC_SYNTHETIC = 4096;
+const int ACC_ANNOTATION = 8192;
+const int ACC_ENUM = 16384;
+const int ACC_MANDATED = 32768;
+const int ACC_MODULE = 32768;
+const int ACC_DEPRECATED = 131072;
+
+
+// jump instruction related constants
+const int JUMP = 0;
```

Comment:
How about using the java byte code spec value `167` here, then we don't need a switch case in java side.